template-notifications.php

```diff
@@ -42,7 +42,7 @@
                                 <div class="expanded small button-group">
                                     <button id="all" type="button"
                                             onclick="toggle_buttons('all'); get_notifications( all = true, true, );"
-                                            class="button hollow"><?php echo esc_html_x( 'All', 'As in all things', 'disciple_tools' ) ?>
+                                            class="button hollow"><?php echo esc_html_x( 'All', 'List Filters', 'disciple_tools' ) ?>
```

Comment:
This does not help clarity

Reply:
> This does not help clarity

Correct - but it does collect this string into all the others so that it doesn't have a unique context.

Reply:
Then maybe we should remove the unhelpful context instead

Reply:
on all of them

Reply:
> Then maybe we should remove the unhelpful context instead

The reason it was put there was that a translator asked about the context because for their language it made a difference.

Reply:
The "As in all things" context?

Reply:
> The "As in all things" context?

Yes that context was added. But actually that question has been answered in poeditor so any future translators will see the answer there anyway So it should be fine to make all these the same. (ie use List Filters)